Sources/Client/Client/Client+Request.swift

```diff
@@ -157,6 +157,8 @@ extension Client {
         
         if let connectionId = webSocket.lastConnectionId {
             queryItems.append(URLQueryItem(name: "client_id", value: connectionId))
+        } else if endpoint.requiresConnectionId {
```

Comment:
Not part of your PR but since you are touching it, let's fix it. We used to support both `client_id` and `connection_id` but now everything works fine with just `connection_id` (`client_id` is used as fallback, that's why current code works).

```swift
queryItems.append(URLQueryItem(name: "client_id", value: connectionId))
```

should be:

```swift
queryItems.append(URLQueryItem(name: "connection_id", value: connectionId))
```